Failed to load the data-explorer package

[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.32.0 x64
**Electron**: 2.0.9
**OS**: Ubuntu 16.04.5
**Thrown From**: [data-explorer](https://github.com/BenRussert/data-explorer) package 0.3.1


### Stack Trace

Failed to load the data-explorer package

```
At Cannot find module '@blueprintjs/select'

Error: Cannot find module '@blueprintjs/select'
    at Module._resolveFilename (module.js:543:15)
    at Module._resolveFilename (/usr/share/atom/resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.get_Module._resolveFilename (/usr/share/atom/resources/app/static/<embedded>:11:154364)
    at Module.require (/app.asar/static/index.js:40:43)
    at require (/usr/share/atom/resources/app/static/<embedded>:11:145671)
    at /packages/data-explorer/node_modules/@nteract/transform-dataresource/lib/VizControls.js:20:15)
    at /packages/data-explorer/node_modules/@nteract/transform-dataresource/lib/VizControls.js:391:3)
    at Module.get_Module._compile (/usr/share/atom/resources/app/static/<embedded>:11:146381)
    at Object.value [as .js] (/usr/share/atom/resources/app/static/<embedded>:11:149928)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
    at Function.Module._load (module.js:496:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (/usr/share/atom/resources/app/static/<embedded>:11:145671)
    at /packages/data-explorer/node_modules/@nteract/transform-dataresource/lib/index.js:48:43)
    at /packages/data-explorer/node_modules/@nteract/transform-dataresource/lib/index.js:495:3)
    at Module.get_Module._compile (/usr/share/atom/resources/app/static/<embedded>:11:146381)
    at Object.value [as .js] (/usr/share/atom/resources/app/static/<embedded>:11:149928)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
    at Function.Module._load (module.js:496:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (/usr/share/atom/resources/app/static/<embedded>:11:145671)
    at /packages/data-explorer/lib/common.js:33:2)
    at /packages/data-explorer/lib/common.js:11:76)
    at Module.get_Module._compile (/usr/share/atom/resources/app/static/<embedded>:11:146381)
    at Object.value [as .js] (/usr/share/atom/resources/app/static/<embedded>:11:149928)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
    at Function.Module._load (module.js:496:3)
```

### Commands

```
```

### Non-Core Packages

```
data-explorer 0.3.1 
Hydrogen 2.6.0 
hydrogen-launcher 1.2.2 
hydrogen-python 0.0.7 
```

```
Error: Cannot find module '@blueprintjs/select'
```
Looks like an issue with the latest release of `@nteract/transform-dataresource`. 

I'll look at this tonight. cc/ @rgbkrk

--- a/package.json
+++ b/package.json
@@ -28,7 +28,7 @@
   },
   "dependencies": {
     "@babel/runtime-corejs2": "^7.0.0",
-    "@nteract/transform-dataresource": "^4.3.2",
+    "@nteract/transform-dataresource": "^5.0.0-alpha.0",
     "lodash": "^4.17.10",
     "react": "^16.4.2",
     "react-dom": "^16.4.2",

